### ISSUE TITLE
Update Docker image

### ISSUE THREAD

**Issue by trashcluster** [jffz/docker-ebot#11]
Hi, is the Docker image up to date with the latest commit ? I'm trying to use the toornament API but i doesn't seem to work. Toornament recently updated to V2 and the ebot repo was updated accordingly and i was wondering if the docker image was also updated. 
Cheers!

**Comment by DrumSlayers** [comment]
The image on Docker Hub isn't up to date, but the Dockerfile is provided so you can make your own updated image.
I will push an updated image on docker hub later if you want

**Comment by DrumSlayers** [comment]
If you try this
https://github.com/DrumSlayers/docker-ebot
It **SHOULD** work
If not, leave an issue, i will look for it.

**Comment by jffz** [comment]
This repo should be up to date as a webhook was in place with the original ebot repository.
I will look into it to bring it back.

**Comment by DrumSlayers** [comment]
Yeah it is, i can build the dockerfile without any problem. The docker hub image with the latest tag isn't up to date, just that.

I originally forked because i have to do customization, due to IP problem with Docker when you host CS:GO container and ebot container on the same docker. CSGO don't communicate to eBot using the external IP but with the internal docker IP (container IP) and eBot reject everything, resulting to commands not working.
